variables.tf

```diff
@@ -214,4 +215,15 @@ locals {
   # note the two white space chars at the start of the line, this corresponds to the
   # formatting in worker-kubelet-conf.yaml and master-kubelet-conf.yaml
   feature_gates_yaml_fragment = "${join("\n  ", formatlist("%s: %s", keys(var.feature_gates), values(var.feature_gates)))}"
+
+  # cluster_dns list formatted for KubeletConfiguration yaml
+  #
+  # example:
+  #  clusterDNS: ${cluster_dns}
+  #  ...
+  #  clusterDNS:
+  #    - "169.254.20.10"
+  #    - "10.3.0.10"
+  #
+  cluster_dns_yaml = "${join("", formatlist("\n  - \"%s\"", var.cluster_dns))}"
```

Comment:
does that makes more sense than changing the format inside the yaml https://github.com/utilitywarehouse/tf_kube_ignition/blob/master/resources/master-kubelet-conf.yaml#L12 
like:
```
clusterDNS: [ "169.254.20.10", "10.3.0.10" ]
```
?

Reply:
I can't print a list and formatting it as a json list is actually worse because you no longer format a single element, but also need to format the start fo the list `[` and the end `]`.

Leaving it like this until re-done in 0.12